content/Services/RHOBS/rules-and-alerting.md

```diff
@@ -54,27 +76,43 @@ groups:
       severity: page
 ```
 
-Now send a `PUT` request to `/api/v1/rules/raw` endpoint, specifying the YAML file, to **create** your alerting rule using the Rules API:
+Now to set this rule file, you can use obsctl,
+
+```bash
+obsctl metrics set --rule.file=/path/to/alerting-rule.yaml
+```
+
+obsctl uses the credentials you saved earlier, to make an authenticated `application/yaml` `PUT` request to the `api/v1/rules/raw` endpoint of the Observatorium API, which **creates** your alerting rule.
+
+obsctl should print out the response, which, if successful, would be: `successfully updated rules file`.
+
+Besides checking this response, you can also list or **read** the configured rules for your tenant by,
 
 ```bash
-curl -X PUT --data-binary @alerting-rule.yaml --header "Content-Type: application/yaml" http://localhost:8080/api/metrics/v1/rhobs/api/v1/rules/raw
+obsctl metrics get rules.raw
 ```
 
-Besides checking if you've got a 200 response, you can also list the rules for the tenant (in this case, `rhobs`):
+This would make a `GET` request to `api/v1/rules/raw` endpoint and return the rules you configured in YAML form. This endpoint will immediately reflect any newly set rules.
+
+Note that in the response a `tenant_id` label for the particular tenant was added automatically. Since Observatorium API is tenant-aware, this extra validation step is also performed. Also, in the case of rules expressions, the `tenant_id` labels are injected into the PromQL query, which ensures that only data from a specific tenant is selected during evaluation.
```

Comment:
Maybe we don't need to go into too much details on label enforcement, since this should primarily be 'how to' tutorial. Maybe just shortening it to explaining why users will see `tenant_id`, to keep the amount of information digestible, something like:

```suggestion
Note that in the response a `tenant_id` label for the particular tenant was added automatically, since this ensures tenancy in Observatorium.
```